strawberry_django/apps.py

```diff
@@ -4,3 +4,6 @@
 class StrawberryDjangoConfig(AppConfig):
     name = "strawberry_django"
     verbose_name = "Strawberry django"
+
+    def ready(self):
+        pass
```

Comment:
Is there something missing from here? Otherwise you might want to remove this

Reply:
Yes, looks like the receivers need to be added still.